src/swarm/client/request/params/IRequestParams.d

```diff
@@ -538,7 +538,6 @@ public abstract class IRequestParams
         ***********************************************************************/
 
         static typeof(&this) serialized ( void[] data )
-        in
```

Comment:
The commit message for this commit (" Move `in` contracts within function bodies ") is a bit inaccurate. Maybe this?
"
Move `in` contracts within function bodies
All `in` contracts were converted into `verify` calls in method bodies in a previous PR. It seems some were missed. Converting these now. (The reason for converting `in` contracts to `verify` calls is to make important checks also occur in release builds.)
"